src/tests/sqlsmith/src/lib.rs

```diff
@@ -87,6 +87,11 @@ struct SqlGenerator<'a, R: Rng> {
     /// We might not read from all tables.
     bound_relations: Vec<Table>,
 
+    /// Relations in generated in 'From' statement shall be added to bound_relations and
+    /// bound_columns after all the relations are generated because they are not useable among
+    /// the relations
+    parallel_relations: Vec<Table>,
```

Comment:
```suggestion
    lateral_contexts: Vec<Table>,
```

To be consistent with naming in `~/projects/risingwave/src/frontend/src/binder/mod.rs::Binder`